project1.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="project1.css" type="text/css">
+  <link href="https://font.googleapis.com/css?family=Lobster" rel="stylesheet" type="text/css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+  integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <link rel="stylesheet" href="https://fonts.sandbox.google.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />
+  <title>Setup and Mobile Version Skeleton</title>
+</head>
+
+<body>
+  <header>
+<div class="grid-container">
+  <div>
+<h1>About Myself</h1>
+
+  <p>I'm a software developer! I can help you build a product, feature or website
+    look through some of my work and experience! If you like what you see and have
+    a project you need coded, don't hesitate to contact me.</p>
+
+    <button class="button">LET'S CONNECT</button>
+
+      <a class="twitter"><img src="twitter.jpg" alt=" "></a>
+      <a class="linkedin"><img src="Linkedinicon.jpg" alt=" "></a>
+      <a class="medium"><img src="medium.jpg" alt=" "></a>
+      <a class="github"><img src="github.jpg" alt=" "></a>
+      <a class="angellist"><img src="angellist.jpg" alt=" "></a>
+
+      <button class="resume">GET MY RESUME</button>
+  
+</div>
+     
+<div>
+        <h5>Languages</h5>
+        <a class="languages"><img src="LanguageList.jpg" alt=" "></a>
+        <h5>Frameworks</h5>
+        <a class="languages"><img src="SeparatorBottom.png" alt=" "></a>
+        <h5>Skills</h5>
+        <a class="languages"><img src="SeparatorBottom.png" alt=" "></a>
+</div>
+</div>
+
+
+
+
+  
+</header>
```

Comment:
One last thing for this section, don't forget to stick to [the Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) and use [flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) to align items! :stuck_out_tongue_winking_eye: